test/hba_test.rules

```diff
@@ -5,10 +5,6 @@
 # hostssl    DATABASE  USER  ADDRESS  METHOD  [OPTIONS]
 # hostnossl  DATABASE  USER  ADDRESS  METHOD  [OPTIONS]
 
-# ws
- 	 	 	
-			# z
-
```

Comment:
Wasn't this here for some parsing test?

Reply:
I couldn't gather that from the commit message that added the file 8 years ago. The lines seemed like nonsense to me and they were causing diff when I saved because they had trailing whitespace that was removed by the editorconfig settings.

I added them back now and disabled the trailing whitespace removal for this file.